lua/stats.lua

```diff
@@ -273,6 +273,7 @@ function mod:newDevRxCounter(name, dev, format, file)
 	obj.sleep = 100
 	setmetatable(obj, devRxCounter)
 	obj:getThroughput() -- reset stats on the NIC
```

Comment:
this call was originally intended to clear the counters (and it worked until we switched to DPDK counters for most NICs which are not clear-on-read), please delete this call